components/Releases/ReleaseLayout.jsx

```diff
@@ -20,67 +20,103 @@ export default function ReleaseLayout({
 }) {
   const [authorized, setAuthorized] = useState(!release.is_password_protected)
   const [releaseDate, setReleaseDate] = useState(new Date(release.release_date))
+  const [isClient, setIsClient] = useState(false)
```

Comment:
What is client?

Reply:
This is how Next docs told me to allow the iframe. Add it to a useEffect to make it load after the page loads.